validator/index.d.ts

```diff
@@ -1,6 +1,6 @@
 // Type definitions for validator.js v5.7.0
```

Comment:
I think yes, but I didn't make more changes to increase version number. I want to do it in the evening.